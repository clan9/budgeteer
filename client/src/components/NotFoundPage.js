import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
  return (
    <div>
      <Link to="/">
        <h3>Sorry, page not found. Click here to go to the home page</h3>
      </Link>
    </div>
  );
};

export default NotFoundPage;
