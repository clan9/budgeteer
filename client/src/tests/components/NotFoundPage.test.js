import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

describe('NotFoundPage component', () => {
  it('should render the not found page correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
