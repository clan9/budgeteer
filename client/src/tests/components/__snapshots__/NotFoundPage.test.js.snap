// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should render the not found page correctly 1`] = `
<div>
  <Link
    to="/"
  >
    <h3>
      Sorry, page not found. Click here to go to the home page
    </h3>
  </Link>
</div>
`;
